packages/@cdktf/provider-generator/lib/get/generator/emitter/resource-emitter.ts

```diff
@@ -52,6 +55,14 @@ export class ResourceEmitter {
     );
   }
 
+  private emitStaticMethods(resource: ResourceModel) {
+    this.code.line(
+      `public static importGenerateConfig(scope: Construct, name: string, id: string, provider?: cdktf.TerraformProvider) {
+        return new cdktf.ImportableResource(scope, name, { terraformResourceType: "${resource.terraformResourceType}", importId: id, provider });
+      }`
```

Comment:
Let's fix the indentation of the generated code 🤓 (Will require the snapshots to be updated 😬 Sorry)
```suggestion
    return new cdktf.ImportableResource(scope, name, { terraformResourceType: "${resource.terraformResourceType}", importId: id, provider });
}`
```